incubator/java-openliberty/image/project/pom.xml

```diff
@@ -0,0 +1,125 @@
+        <plugins>
+            <!-- Enforcing OpenLiberty and JDK Version -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>3.0.0-M2</version>
+                <executions>
+                <execution>
+                    <id>enforce-versions</id>
+                    <goals>
+                        <goal>enforce</goal>
+                    </goals>
+                    <configuration>
+                        <rules>
+                            <requireJavaVersion>
+                                <version>[1.8,1.9)</version>
+                            </requireJavaVersion>
+                            <requireProperty>
+                                <property>version.openliberty-runtime</property>
+                                <regex>19.0.0.12</regex>
+                                <regexMessage>OpenLiberty runtime version must be 19.0.0.12</regexMessage>
+                            </requireProperty>
+                        </rules>
+                    </configuration>
+                </execution>
+            </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <!--
+            Note for this stack, there are three "modes" through the POMs:
+            In addition to:
+               1) the standard appsody "local dev" mode, (activated by the presence of the env var: APPSODY_DEV_MODE)
+               2) the `appsody build` Docker biuld
+
+            there is also the possiblity to run from:
+               3) the host system ("natively" in the sense of not using appsody at all).
+
+            It is tempting to consider adding profile-specific behavior for 2) and 3)
+            as we add the profile for 1), but perhaps this should be
+            discussed more within the appsody community before going this
+            far for this particular stack.
+        -->
+        <profile>
+            <id>dev</id>
+            <activation>
+              <property>
+                <name>env.APPSODY_DEV_MODE</name>
+              </property>
+            </activation>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>io.openliberty.tools</groupId>
+                            <artifactId>liberty-maven-plugin</artifactId>
+                            <version>${version.liberty-maven-plugin}</version>
+                            <configuration>
+                                <assemblyArtifact>
+                                    <groupId>${liberty.groupId}</groupId>
+                                    <artifactId>${liberty.artifactId}</artifactId>
+                                    <version>${version.openliberty-runtime}</version>
+                                    <type>zip</type>
+                                </assemblyArtifact>
```

Comment:
Do we need the assemblyArtifact?

Reply:
I think it's helpful to install this in the base, stack image so the user doesn't have to wait as long on `appsody run` to install it.

Reply:
Thanks, Scott.  We can update later to use the latest way to specify the runtime to install.